bungee/src/main/java/us/myles/ViaVersion/bungee/handlers/BungeeChannelInitializer.java

```diff
@@ -24,7 +25,7 @@ public BungeeChannelInitializer(ChannelInitializer<Channel> oldInit) {
     @Override
     protected void initChannel(Channel socketChannel) throws Exception {
         if (!socketChannel.isActive()) {
-            return;
+            throw new CancelException();
```

Comment:
This exception won't be handled correctly because ViaVersion handler isn't in the pipeline

Reply:
So we have to use return there?

Reply:
Resolved in 2c06a42